outlook/indico_outlook/plugin.py

```diff
@@ -151,7 +178,10 @@ class OutlookPlugin(IndicoPlugin):
         'max_event_duration': TimedeltaConverter
     }
     default_user_settings = {
-        'enabled': True,  # XXX: if the default value ever changes, adapt `get_participating_users`!
+        'enabled': True,  # XXX: if the default value ever changes, adapt `get_registered_users`!
+        'registered': True,
+        'favorite_events': True,
+        'favorite_categories': True,
```

Comment:
```suggestion
        'favorite_events': False,
        'favorite_categories': False,
```

For events I'm unsure (defaulting to `False` avoids surprises for users), but categories certainly need to be opt-in.

Reply:
I'm concerned about the discoverability of this feature, if it's disabled by default nobody will find it and it won't be very useful.